### PR TITLE
Feature ge mapobject model

### PR DESCRIPTION
**Pull request by denisfruza**
MapObject model and its related classes.

### CODE REVIEW

File: WPFHospitalEditor/MapObjectModel/MapObject.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        public void setTextBlockPositionOnMap(MapObjectCoordinates mapObjectCoordinates)
+        {
+            this.name.SetValue(Canvas.LeftProperty, mapObjectCoordinates.X);
+            this.name.SetValue(Canvas.TopProperty, mapObjectCoordinates.Y);
+        }
+
+        public void setMapObjectColor()
+        {
+            MapObjectColors mapObjectColors = MapObjectColors.getInstance();
+            this.rectangle.Fill = MapObjectColors.mapObjectTypesColors[MapObjectType];
```
Comment (hi-imAndy) [inline WPFHospitalEditor/MapObjectModel/MapObject.cs:64]: Singletone should always be used with getInstance(). You must not create a new instance of a singletone class.

Comment (denisfruza) [reply WPFHospitalEditor/MapObjectModel/MapObject.cs]: fixed

**Review by hi-imAndy — DISMISSED** [review]

**Review by BoKna031 — APPROVED** [review]
change the current color loading to be done independently of the code as soon as possible

**Review by hi-imAndy — APPROVED** [review]

**Comment by BoKna031** [comment]
[(0)PR Feature ge mapobject model](https://trello.com/c/0MDztX5z/70-0pr-feature-ge-mapobject-model)

**Pull request merged**